dir.props

```diff
@@ -69,14 +69,16 @@
   <!-- Provides package dependency version properties and verification/auto-upgrade configuration -->
   <Import Project="$(MSBuildThisFileDirectory)dependencies.props" />
 
-  <!-- Explicitly setting
-       - SupportsUWP to true when TargetGroup is uap101*. This will enable the correct pinvoke checker to be picked
-       - BlockReflectionAttribute to true. This will enable injecting BlockReflectionAttribute.cs into the compile target.
-       TODO: Both of these properties are being set in buildtools targets, however they need to be updated to reflect uap101 changes.
-  -->
-  <PropertyGroup Condition="'$(TargetGroup)' == 'uap101' or '$(TargetGroup)' == 'uap101aot'">
+  <!-- Only if we're building a binary flavor that's only ever going to run on uapaot is OK for it to have #if NETNATIVE blocks -->
+  <PropertyGroup Condition="'$(TargetGroup)' == 'uapaot'">
+    <DefineConstants>$(DefineConstants),NETNATIVE</DefineConstants>
```

Comment:
I have reviewed all places that use `#if NETNATIVE` and `#if !NETNATIVE` in the current tree. None of them has anything to do with AOT. They are used for UAP-specific code and hacks in the implementation of `System.Security.Cryptography`; `System.Net` and `System.Threading` namespaces. https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/project-guidelines.md (very end) says to use `#if uap` in this case. This ifdef should be defined locally in the few project that need it, and it should be defined for both uap and uapaot TargetGroups so that you get the same behavior both CoreCLR and ProjectN runtimes.